packages/common/src/components/Footer/FooterCookiesModal.tsx

```diff
@@ -14,8 +14,17 @@ const ExternalLinkIconImg = styled.img`
   position: relative;
 `
 
+const StyledTextWithInfoModal = styled(TextWithInfoModal)`
```

Comment:
How is this change realted to this Jira?

Reply:
It's not, but the arrow used to be there before and has somehow disappeared. So I thought the quickest way was to just put it back again, instead of creating a new ticket in Jira etc.